fs/vfs/fs_epoll.c

```diff
@@ -36,23 +36,45 @@
 #include <nuttx/clock.h>
 #include <nuttx/fs/fs.h>
 #include <nuttx/kmalloc.h>
+#include <nuttx/list.h>
 #include <nuttx/mutex.h>
+#include <nuttx/signal.h>
 
 #include "inode/inode.h"
 
 /****************************************************************************
  * Private Types
  ****************************************************************************/
 
-struct epoll_head
+typedef struct epoll_node_s
 {
-  int size;
-  int occupied;
-  int crefs;
-  mutex_t lock;
-  FAR epoll_data_t *data;
-  FAR struct pollfd *poll;
-};
+  struct list_node      node;
+  epoll_data_t          data;
+  struct pollfd         pfd;
+} epoll_node_t;
```

Comment:
I'm not sure about the style here. In many places I see stye like
```
struct epoll_node_s
{
  struct list_node      node;
  epoll_data_t          data;
  struct pollfd         pfd;
};

typedef struct epoll_node_s epoll_node_t;
```

Reply:
You are right, i will update.

Reply:
Done